Source/VOE/VOE.csproj

```diff
@@ -76,5 +75,8 @@
     <Compile Include="Outpost_Town.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <None Include="packages.config" />
```

Comment:
Can you use `<PackageReference>` instead of `packages.config`?

Reply:
Done, was lazy with setup because at first thought I was just tweaking something for myself :P